src/main/java/eu/deltacraft/deltacraftteams/listeners/CraftItemListener.kt

```diff
@@ -16,7 +16,33 @@ class CraftItemListener(private val pointsQueue: PointsQueue) : Listener {
 
     companion object {
         val map = hashMapOf<Material, Int>(
-            // Material.BEACON to 5, // TODO: Fill from excel
+            // BLOCKS
+            // -------
+            Material.BEACON to 35,
+            Material.ENCHANTING_TABLE to 10,
+            Material.END_CRYSTAL to 20,
+            Material.ENDER_CHEST to 20,
+            // -------
+            // ITEMS
+            // -------
+            Material.GOLDEN_APPLE to 10,
+
+            // -------
+            // TOOLS
+            // -------
+            // Swords
+            Material.STONE_SWORD to 2,
+            Material.GOLDEN_SWORD to 4,
+            Material.IRON_SWORD to 4,
+            Material.DIAMOND_SWORD to 15,
+            Material.NETHERITE_SWORD to 20,
```

Comment:
Netherite asi nepůjde ne? Ten se necraftí, ale "upgraduje" ve "Smithing Table". Nutné otestovat

Reply:
No jo! To je pravda!